test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -561,6 +561,43 @@ foo
             }
         }
     }
+
+    Context "-WorkingDirectory parameter" {
+        BeforeAll {
+            $folderName = (New-Guid).ToString();
+            New-Item -Path ~/$folderName -ItemType Directory
+            $ExitCodeBadCommandLineParameter = 64
+        }
+
+        AfterAll {
+            Remove-Item ~/$folderName -Force -ErrorAction SilentlyContinue
+        }
+
+        It "Can set working directory to '<value>'" -TestCases @(
```

Comment:
It would be good to have a case where $value has a space in directory name.
Also a case for Windows-style path separators in directory path.

Reply:
Will add those cases